docs/src/docs/common-pitfalls.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Common pitfalls
+---
+
+For the most part, you should be able to seamlessly migrate your site from 11ty to Slinkity with minimal reconfiguration. But there are a few changes that may catch you off guard. These gotchas are summarized below.
+
+## Certain files disappeared from my production build output folder
+
+In an 11ty site, you are free for the most part to structure your project however you like. You can put your static assets in a `public` folder outside your [11ty input directory]((https://www.11ty.dev/docs/config/#input-directory)) and pass-through copy them, but you don't need to—you could also store all of your assets inside your 11ty input directory. It doesn't matter which approach you take, so long as you configure your 11ty project accordingly and write your files to the correct locations. However, you need to take a slightly different approach in Slinkity due to the standards enforced by Vite.
+
+First, we need to understand that a Slinkity build consists of multiple steps:
+
+1. Processing your site with 11ty and dumping the result into a temporary build folder (e.g., `.11ty-build-DcNVBN`).
+2. Processing the temporary build folder with Vite to bundle assets on the pages that reference them.
+3. Writing the temporary build folder to your 11ty output directory (e.g., `_site`)
+4. Deleting the temporary build folder.
+
+Vite is very strict about where it wants you to place your static assets—in a [dedicated `public` folder](https://vitejs.dev/guide/assets.html#the-public-directory). This allows it to distinguish static assets from ones that are referenced by your pages (like stylesheets) and to bundle the referenced assets while leaving the static ones alone. Any static asset that isn't referenced by a page and isn't in the `public` directory will be removed from the final build output. For this reason, you may find that certain files disappear from production builds, even though they were technically processed correctly by 11ty.
+
+Below are some scenarios where you may encounter this pitfall:
+
+- Self-hosted font files
+- Static images (both processed and passthrough-copied)
+- Miscellaneous static files (`sitemap.xml`, `robots.txt`, and the like)
+
+Let's look at an example.
+
+### The old approach (11ty)
+
+Suppose you're using 11ty to auto-generate a `sitemap.xml` for your site. In 11ty, you'd create this file using your chosen templating language extension and set a permalink in its front matter relative to the output directory, like so:
+
+{% raw %}
+```liquid
+{% comment %}src/sitemap.liquid{% endcomment %}
```

Comment:
```suggestion
{% comment %}src/sitemap.liquid{% endcomment %}
<!--src/sitemap.liquid-->
```

I don't think we're using the `comment` shortcode in our project. Is this a liquid feature? We're admittedly using nunjucks as our default language for `.md` files, so this shortcode doesn't get resolved.

Reply:
Yeah, this is a Liquid feature. I wanted to clarify the file's name using comments, but Prism didn't seem to like either `njk` or `nunjucks` as the language name, so I just went with Liquid.